lib/swaggerUtils/swaggerToOpenapi.js

```diff
@@ -0,0 +1,21 @@
+const Swagger2OpenAPI = require('swagger2openapi');
+
+module.exports = {
+  convertSwaggerToOpenapi: function(parsedSwagger) {
+    try {
+      return Swagger2OpenAPI.convertObj(
+        parsedSwagger,
+        {
+          fatal: false,
+          patch: true,
+          anchors: true,
+          warnOnly: true
+        }
+      );
+    }
+    catch (error) {
+      return error;
```

Comment:
Shouldn't this be `throw` and not return? How do we identify the error vs result if we return both?

Reply:
Also, can we introduce test-case for this when there is an error?